spec/support/acts_as_tenant.rb

```diff
@@ -38,7 +44,7 @@ module FeatureHelpers
     #                        tests. The tenant must have been set using a let construct.
     # @param proc [Proc] The block containing the test definitions.
     def with_tenant(tenant, &proc)
-      super(tenant) do
+      super(tenant) do |*params|
```

Comment:
Just to be sure. I don't have any params, but since I do not use it, I should pass it on...